cloud_experts_tutorials/cloud-experts-custom-dns-resolver.adoc

```diff
@@ -0,0 +1,259 @@
+
+. Before the cluster is created, configure your DNS server to forward all DNS requests for `<cluster-name>.hypershift.local` to your Amazon Route 53 Inbound Resolver endpoints. For BIND DNS servers, edit your `/etc/named.conf` file in your favorite text editor and add a new zone using the below example: 
++
+.Example
+[source,terminal]
+----
+zone "<cluster-name>.hypershift.local" { <1>
+  type forward;
+  forward only;
+  forwarders { <2>
+    10.0.45.253;
+    10.0.23.131;
+    10.0.148.159;
+  };
+};
+----
+<1> Replace `<cluster-name>` with your ROSA HCP cluster name.
+<2> Replace with the IP addresses of your inbound resolver endpoints collected above, ensuring that following each IP address there is a `;`.
++
+. xref:../rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-account-wide-sts-roles-and-policies_rosa-hcp-sts-creating-a-cluster-quickly[Create your cluster].
++
+. Once your cluster has begun the creation process, locate the newly created private hosted zone:
++
+[source,terminal]
+----
+$ aws route53 list-hosted-zones-by-vpc \
+  --vpc-id ${VPC_ID} \
+  --vpc-region ${REGION} \
+  --query 'HostedZoneSummaries[*].Name' \
+  --output table
+----
++
+.Example output
++
+[source,text]
+----
+--------------------------------------------------
+|             ListHostedZonesByVPC               |
++------------------------------------------------+
+|  rosa.domain-prefix.lkmb.p3.openshiftapps.com. |
+|  cluster-name.hypershift.local.                |
++------------------------------------------------+
+----
++
+[NOTE]
+====
+It may take a few minutes for the cluster creation process to create the private hosted zones in Route 53. If you do not see an `p3.openshiftapps.com` domain, wait a few minutes and run the command again.
+====
++
+. Once you know the unique ID of the cluster domain, configure your DNS server to forward all DNS requests for `rosa.<domain-prefix>.<unique-ID>.p3.openshiftapps.com` to your Amazon Route 53 Inbound Resolver endpoints. For BIND DNS servers, edit your `/etc/named.conf` file in your favorite text editor and add a new zone using the below example: 
++
+.Example
+[source,terminal]
+----
+zone "rosa.<domain-prefix>.<unique-ID>.p3.openshiftapps.com" { <1>
+  type forward;
+  forward only;
+  forwarders { <2>
+    10.0.45.253;
+    10.0.23.131;
+    10.0.148.159;
+  };
+};
+----
+<1> Replace `<domain-prefix>` with your cluster domain prefix and `<unique-ID>` with your unique ID collected above.
+<2> Replace with the IP addresses of your inbound resolver endpoints collected above, ensuring that following each IP address there is a `;`.
+
+Your cluster is now configured to use a custom domain name server for name resolution.
```

Comment:
Text in a `.Procedure` section doesn't fit our templates. If you need to have something here to help the user know that they are done, consider adding a `.Verification` section with the step(s) to confirm success. Otherwise, this type of statement is unneeded.
```suggestion
```